src/components/chat/comment-card.test.tsx

```diff
@@ -0,0 +1,49 @@
+import { render, screen } from "@testing-library/react";
+import { Provider } from "mobx-react";
+import React from "react";
+import { UserModelType } from "../../models/stores/user";
+import { ENavTab } from "../../models/view/nav-tabs";
+import { CommentCard } from "./comment-card";
+
+describe("CommentCard", () => {
+
+  const stores = { ui: { activeNavTab: ENavTab.kMyWork } };
+  const activeNavTab = stores.ui.activeNavTab;
+  const testUser  = {  id: "0",
+                      name: "Test Teacher",
+                    } as UserModelType;
```

Comment:
Opinions vary on this, but I would probably format this like this:
```suggestion
  const testUser = { id: "0", name: "Test Teacher" } as UserModelType;
```